aces/Loris
# [jsx/MultiSelectDropdown.js] Close multiSelectDropdown when user clicks outside the dropdown

## Brief summary of changes

When clicking outside the multiSelectDropdown the dropdown should close. This PR adds that functionality.

#### Testing instructions (if applicable)

1. Visit DQT.
2. Define Fields
3. Click on multiselectdropdown "None selected"
4. Click outside dropdown and it should close.
